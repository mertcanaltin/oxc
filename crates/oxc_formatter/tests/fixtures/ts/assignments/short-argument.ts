const requestTrie =
  TernarySearchTree.forPaths<IRecursiveWatchRequest>(!isLinux);
