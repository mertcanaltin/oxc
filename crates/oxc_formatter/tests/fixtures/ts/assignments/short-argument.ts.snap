---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
const requestTrie =
  TernarySearchTree.forPaths<IRecursiveWatchRequest>(!isLinux);

==================== Output ====================
const requestTrie =
  TernarySearchTree.forPaths<IRecursiveWatchRequest>(!isLinux);

===================== End =====================
